app/views/home/index.html.erb

```diff
@@ -1,17 +1,21 @@
 <main class="home">
-  <header class="home__header"><a href="#" class="home__logo">ground rules</a></header>
+<div class="home__inner">
+  <header class="home__header"><h1><a href="#" class="home__logo">ground rules</a></h1></header>
```

Comment:
Break this stuff on multiple lines.  I didn't even see the `<h1>` in there the first couple of passes.

Also -- that `href` should be `href="/"`